### PR TITLE
Create Back-Button as a common component

### PR DESCRIPTION
**Pull request by AlaaTaima**
Relates #56

### CODE REVIEW

File: src/components/BackButton/backbutton.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.backButton {
+    color: rgba(36, 36, 36, 0.6);
+    font-size: 20px;
+    width: 60px;
+}
```
Comment (yosefanajjar) [inline src/components/BackButton/backbutton.css:7]: add new line

Comment (AlaaTaima) [reply src/components/BackButton/backbutton.css:7]: Done.

**Review by fares98 — APPROVED** [review]

File: src/components/BackButton/backbutton.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.backButton {
+    color: rgba(36, 36, 36, 0.6);
+    font-size: 20px;
+    width: 60px;
```
Comment (Fatmasiam) [inline src/components/BackButton/backbutton.css]: don't specify the width for the back button div 
and trying to add 
```css
 cursor: pointer;
```

File: src/components/BackButton/index.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Icon } from 'antd';
+import './backbutton.css';
+
+const BackButton = () => {
+  return (
+    <div className="backButton">
```
Comment (Fatmasiam) [inline src/components/BackButton/index.jsx]: how can we implement the back button?
i mean where we should take action to it to back to the previous page

File: src/components/BackButton/backbutton.css (hunk 2 above)
Comment (AlaaTaima) [reply src/components/BackButton/backbutton.css]: Done

**Review by fares98 — DISMISSED** [review]

**Review by alaa-yasin — DISMISSED** [review]

File: src/components/BackButton/index.jsx (hunk 3 above)
Comment (yosefanajjar) [inline src/components/BackButton/index.jsx]: we don't use camel case in classes naming so you can name it back-button instead

Comment (AlaaTaima) [reply src/components/BackButton/index.jsx]: ok , done

Comment (AlaaTaima) [reply src/components/BackButton/index.jsx]: Done

**Review by Fatmasiam — APPROVED** [review]

**Pull request merged**